app/utils/skills-list.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+const {
+  get
+} = Ember;
+
+let skillsList = {
```

Comment:
I love that this is split out into a regular module, rather than a mixin/service.

As for the conventions, did you mean you are not sure if you should be exporting just an object literal? For this use case, where you are exporting 2 functions, I don't really see a problem with it. 

you *could* export them both as named exports and let things that are importing it choose what they import. Or even make the default export be an object literal of the two functions still.